src/gdft.c

```diff
@@ -1293,25 +1294,43 @@ BGD_DECLARE(char *) gdImageStringFTEx (gdImage * im, int *brect, int fg, const c
 			any2eucjp (tmpstr, string, BUFSIZ);
 			next = tmpstr;
 		} else {
-			next = string;
+			next = (char *)string;
```

Comment:
something is broken if we're casting the const away.  please undo these changes.